bundles/org.openhab.automation.pythonscripting/pom.xml

```diff
@@ -94,6 +109,13 @@
           <pmdFilter>${project.basedir}/suppressions.properties</pmdFilter>
         </configuration>
       </plugin>
+      <!-- workaround for failing clean of checked out git repo breaking the Windows build, #19250 -->
```

Comment:
Thanks if this works, this has been quite annoying, since I've had to manually delete the target folder every time.